packages/app/src/cli/services/generate.ts

```diff
@@ -94,13 +93,9 @@ function checkLimits(
 
 function limitReached(app: AppInterface, specifications: ExtensionSpecification[], templateType: TemplateType) {
   const type = templateType.type
-  if (type === 'function') {
-    return app.extensions.function.length >= blocks.functions.defaultRegistrationLimit
-  } else {
```

Comment:
There is a `function` spec already, we don't need to treat it as a special case anymore